stix_shifter_modules/cybereason/README.md

```diff
@@ -0,0 +1,1323 @@
+# Cybereason Connector
+
+```
+I.    CYBEREASON API ENDPOINTS
+II.   PATTERN EXPRESSION WITH STIX ATTRIBUTES - SINGLE OBSERVATION
+III.  PATTERN EXPRESSION WITH STIX ATTRIBUTES - MULTIPLE OBSERVATION
+IV.   PATTERN EXPRESSION WITH STIX ATTRIBUTES - EXECUTE QUERY
+V.    PATTERN EXPRESSION WITH STIX ATTRIBUTES - PING QUERY
+VI.   LIMITATIONS
+VII.  OBSERVATIONS
+VIII. REFERENCES
+```
```

Comment:
you should create a proper mark up table of contents since this is a a long readme. here's an example how to do it: https://raw.githubusercontent.com/opencybersecurityalliance/stix-shifter/develop/OVERVIEW.md

Reply:
A table of contents has been created in readme.